### PR TITLE
Fix parted disk error without mklabel

### PR DESCRIPTION
**Pull request by chunfuwen**
Signed-off-by: chunfuwen <chwen@redhat.com>
Address issue:CmdError: Command 'parted -s /dev/sdb print' failed (rc=1)&#10;
Jira link:https://projects.engineering.redhat.com/browse/LIBVIRTAT-1341

### CODE REVIEW

**Comment by chunfuwen** [comment]
@will-Do 
@waynesun09 

Please take a look

File: libvirt/tests/src/virtual_disks/virtual_disks_multivms.py [hunk 1]
```diff
@@ -142,6 +142,7 @@ def get_vm_disk_xml(dev_type, dev_name, **options):
                 is_setup=True, is_login=True, image_size=image_size)
             logging.debug("iscsi dev name: %s", disk_source)
             # Format the disk and make the file system.
+            libvirt.mk_label(disk_source)
```
Comment (balamuruhans) [inline libvirt/tests/src/virtual_disks/virtual_disks_multivms.py]: `libvirt.mk_part()` is already taking care of the mklabel right?

Comment (chunfuwen) [inline libvirt/tests/src/virtual_disks/virtual_disks_multivms.py]: libvirt.mk_part() introduce some changes that mklabel need to be explicitly called before that

Comment (balamuruhans) [inline libvirt/tests/src/virtual_disks/virtual_disks_multivms.py]: fine if it expected to be called explicitly, then we have to consider making changes in other files `resource_abnormal.py`, `virtual_disks_multidisks.py` and `virtual_disks_encryption.py` which uses `libvirt.mk_part()`

Comment (chunfuwen) [inline libvirt/tests/src/virtual_disks/virtual_disks_multivms.py]: Fixed issues in resource_abnormal.py, virtual_disks_multidisks.py and virtual_disks_encryption.py also.

File: libvirt/tests/src/virtual_disks/virtual_disks_encryption.py [hunk 2]
```diff
@@ -139,6 +139,7 @@ def check_in_vm(vm, target, old_parts):
                 logging.error("Cann't see added partition in VM")
                 return False
 
+            libvirt.mk_label("/dev/%s" % added_part)
```
Comment (balamuruhans) [inline libvirt/tests/src/virtual_disks/virtual_disks_encryption.py]: instead forming the strings in many places, we can have a variable same as in other files,
```
device_source = os.path.join(os.sep, dev, added_part)
libvirt.mk_label(device_source)
libvirt.mk_part(device_source)
```

Comment (chunfuwen) [inline libvirt/tests/src/virtual_disks/virtual_disks_encryption.py]: updated

**Comment by will-Do** [comment]
Thanks @chunfuwen @balamuruhans

**Pull request merged**